showcase/app/components/page-components/form/file-input/sub-sections/base-control.gts

```diff
@@ -37,23 +38,23 @@ const SubSectionBaseControl: TemplateOnlyComponent<SubSectionBaseControlSignatur
 
       <ShwTextH3>States</ShwTextH3>
 
-      <ShwFlex @gap="2rem" as |SF|>
+      <ShwGrid @columns={{3}} @gap="2rem" as |SG|>
```

Comment:
Changing this to a Grid allowed the style update to be tested & seen in the example when the browser screen width was reduced. (Let me know if you think it would be better to add a new example.)